simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java

```diff
@@ -4,66 +4,77 @@
 import com.evbox.everon.ocpp.simulator.station.actions.Plug;
 import com.evbox.everon.ocpp.simulator.station.actions.Unplug;
 import com.evbox.everon.ocpp.simulator.station.actions.UserMessage;
-import com.google.common.base.Preconditions;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Stream;
 
+import static com.google.common.base.Preconditions.checkArgument;
+
 /**
  * Contains list of console commands representing user actions applicable to certain station.
  * For each command there is a validation logic and mapping to {@link UserMessage}
  */
 public enum ConsoleCommand {
 
     PLUG {
+        private static final int EVSE_ID_INDEX = 0;
+        private static final int CONNECTOR_ID_INDEX = 1;
+
         @Override
         public UserMessage toUserMessage(List<String> args) {
             validateArgs(args);
-            return new Plug(Integer.valueOf(args.get(0)));
+            return new Plug(Integer.valueOf(args.get(EVSE_ID_INDEX)), Integer.valueOf(args.get(CONNECTOR_ID_INDEX)));
         }
 
         @Override
         void validateArgs(List<String> args) {
-            validateLength(args, 1);
+            validateLength(args, 2);
 
-            int connectorIdArgIndex = 0;
-            validateNumeric(connectorIdArgIndex, args.get(connectorIdArgIndex));
+            validateEvseAndConnector(args, EVSE_ID_INDEX, CONNECTOR_ID_INDEX);
         }
     },
 
     UNPLUG {
+        private static final int EVSE_ID_INDEX = 0;
```

Comment:
Please move these constants to ConsoleCommand class.

Reply:
It's different for AUTH command so it wouldn't work